### DISCUSSION TITLE
perbedaan jumlah row

### DISCUSSION THREAD

**Discussion by helmihasibuan** [teamalgoritma/community#636] [Q&A] [answered]
Dear team algoritma / bapak / ibu,

Saya mau bertanya adakah referensi untuk menyelesaikan error ini?

![image](https://user-images.githubusercontent.com/65268735/232188597-7f76aa48-b1ca-475c-9239-dd62872ef2b9.png)

Terima Kasih

**Comment by helmihasibuan** [comment, marked as answer]
maaf tidak jadi saya sudah menemukan jawabannya

**Reply by amaliasf** [reply]
Baik, terimakasih untuk konfirmasinya Pak @helmihasibuan
